### DISCUSSION TITLE
SSH menu disappeared for PostgreSQL  connection

### DISCUSSION THREAD

**Discussion by samulenti** [dbeaver/dbeaver#18895] [Q&A] [answered]
### Description

I am no longer able to access the SSH menu to create a tunnel to a remote DB

### DBeaver Version

Community edition 22.3.2

### Operating System

Ubuntu 22.04

### Database and driver

Database: PostgreSQL 12.10
Driver: PostgreSQL JDBC 42.5.0

### Steps to reproduce

1. Edit connection (for a PostgreSQL connection)
2. Connection settings > Initialization Menu
3. There is no tab SSH

**Comment by E1izabeth** [comment]
Hi, @samulenti. It's still here for me as usual (Connection Settings -> SSH, not under Initialization Menu)
![image](https://user-images.githubusercontent.com/28875055/212896360-53c3f255-f137-4691-a9c5-29c957259cfd.png)

**Comment by samulenti** [comment]
Thanks @E1izabeth, you are right, the tab should be under Connection Settings and not under Initialization Menu, but I've not it. I had it for sure with an old version (I don't remember which one it was), but after the update to the last version it is not there anymore...
![image](https://user-images.githubusercontent.com/12124932/212951899-bd47f4d7-bdcc-42f8-b37d-27a2cbae59b0.png)

**Comment by ShadelessFox** [comment]
Hello @samulenti,

Could you please send me the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/.

**Comment by samulenti** [comment]
I have an update on this and unfortunately, I no longer have the logs available.
I installed the package via `apt-get` and updated to the latest version via `apt update/apt upgrade` and I don't know if for some reason there was some problem during the upgrade.
Now I removed the package and reinstalled via `snap`, and the behavior is as expected: I have the SSH menu available again.

Thanks, and sorry for the inconvenience

**Comment by E1izabeth** [comment]
Thank you for response. Then I'll close this issue for now. If it will occur again, inform us please.

**Comment by WhisperWind22** [comment]
Description
I am no longer able to access the SSH menu to create a tunnel to a remote DB
<img width="888" alt="image" src="https://user-images.githubusercontent.com/12683102/215095560-9fd1d62e-c395-4472-8f94-0eeaeaf113a2.png">

[dbeaver-debug.log](https://github.com/dbeaver/dbeaver/files/10519130/dbeaver-debug.log)
Started after updating from version 22.2.2
When I rolled back to version 22.2.2 the tab ssh comes back.
<img width="883" alt="image" src="https://user-images.githubusercontent.com/12683102/215095274-32e5a8f6-547f-4bfe-ad79-26ae29f4a2f8.png">


DBeaver Version
Community edition 22.3.3

Operating System
MacOS Ventura 13.1


Database and driver
Database: PostgreSQL
Driver: PostgreSQL JDBC 42.5.0

Steps to reproduce
New connection
PostgreSQL connection > Next
There is no tab SSH
Additional context
No response

**Comment by E1izabeth** [comment]
@WhisperWind22 could you show me information about installed plugins in Help->Installation Information->Plugins for the DBeaver version where you can reproduce this bug?
Like this
![image](https://user-images.githubusercontent.com/28875055/215100863-a23d5dd2-1db7-40d5-a286-b7cdb07c6068.png)

**Comment by WhisperWind22** [comment]
Sorry, didn't notice the ssh filter

<img width="1189" alt="image" src="https://user-images.githubusercontent.com/12683102/215102735-f03fd0a1-b81b-4ef1-9a6b-16f83e8f5e9e.png">

**Comment by E1izabeth** [comment]
Thank you

**Comment by serge-rider** [comment]
Possibly a side effect of https://github.com/dbeaver/dbeaver/issues/17649

**Comment by ShadelessFox** [comment, marked as answer]
Hello @WhisperWind22,

Make sure this checkbox is **not checked** in the settings of the driver you're using:

![java_oA2XGjCAVb](https://user-images.githubusercontent.com/35821147/215108813-ca139f10-d975-4781-a6a0-9ddeadf1a9d2.png)

Please let me know if it helps.

**Comment by WhisperWind22** [comment]
Hello @ShadelessFox ,

Yes, it helped. The ssh tab appeared when creating a connection and editing existing ones. Thank you!
